packages/bitcore-wallet-service/src/fiatrateservice/fiatrateservice.ts

```diff
@@ -1,5 +1,6 @@
 #!/usr/bin/env node
-const config = require('../config');
+
+var config = require('../config');
```

Comment:
why no `const`?

Reply:
For some reason, if I use `const` I get a typescript error .. something like "You can not declare config 2 times", taking into account the config of fiatrateservice and emailservice
I'm not sure why it compares the declaration of 2 different files